### DISCUSSION TITLE
calculate interface normals based on FeatureFloodingCount

### DISCUSSION THREAD

**Discussion by BoZeng1997** [idaholab/moose#23927] [Q&A General] [answered]
Hi,

Does MOOSE has objects that work together with FeatureFloodingCount to compute the normals of the interfaces between the regions identified?

**Comment by GiudGiud** [comment]
@permcody @laagesen on FeatureFloodingCount

**Reply by laagesen** [reply, marked as answer]
To my knowledge, no we do not have that capability, but @permcody would be the best person to answer. You may be able to find interfacial normals using Paraview, you would create a contour of the isosurface of an order parameter by choosing a certain value to threshold at, similar to how you would choose it in `FeatureFloodCount`.
